packages/hash/frontend/src/pages/signup.page/account-setup.tsx

```diff
@@ -158,7 +158,7 @@ export const AccountSetup: FunctionComponent<AccountSetupProps> = ({
                       : "text-black text-opacity-60 text-xs"
                   }`}
                 >
-                  {shortnameError || (
+                  {(shortnameError === false ? null : shortnameError) ?? (
```

Comment:
Optional: this is ugly and hard to read, maybe we could update `getShortnameError` instead to return `isTouched ? msg : null` rather than `isTouched && msg` to avoid doing it